source/features/shorten-links.tsx

```diff
@@ -5,19 +5,18 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import {linkifiedURLClass} from '../github-helpers/dom-formatters';
+import {codeElementsSelectors} from './show-whitespace';
 
 function init(): void {
 	observe(`a[href]:not(.${linkifiedURLClass})`, {
 		constructor: HTMLAnchorElement,
 		add(link) {
-			// Don't shorten links in code or code suggestions (but shorten them in review comments)
-			// Code explanation: Exclude links if the closest element found is not `.comment-body`
+			// Exclude the link if the closest element found is not `.comment-body`
+			// This avoids shortening links in code and code suggestions, but still shortens them in review comments
 			// https://github.com/refined-github/refined-github/pull/4759#discussion_r702460890
-			if (link.closest('.blob-code, .comment-body, .js-suggested-changes-blob')?.matches('.blob-code, .js-suggested-changes-blob')) {
-				return;
+			if (link.closest(`${codeElementsSelectors}, .js-suggested-changes-blob, .comment-body`)?.classList.contains('comment-body')) {
```

Comment:
The only real change is the addition of the `codeElementSelectors`. Everything else is just refactoring.

Reply:
Can you move this selector to its own array (plus .join())? Maybe with comments.

Reply:
I've dropped `.js-suggested-changes-blob` which was unnecessary. Now it's just `codeElementSelectors` which is documented in `show-whitespace`.

Reply:
> The only real change is the addition of the `codeElementSelectors`.

I didn't quite follow #4759 but it seems this should have been done in that PR 😅

> I've dropped `.js-suggested-changes-blob` which was unnecessary (because superseded by `.blob-code-inner`). Now it's just `codeElementSelectors` which is documented in `show-whitespace`.

Ha! That is essentially reverting [`930f860` (#4759)](https://github.com/refined-github/refined-github/pull/4759/commits/930f860f7d76152d6f944b9c1e66cb980eabcd7f).